lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -657,13 +659,15 @@ sub _create_clone_with_child ($res, $clones, $c, $dependency) {
     $res->children->find_or_create({child_job_id => $c, dependency => $dependency});
 }
 
-sub _create_clones ($self, $jobs, @clone_args) {
+sub _create_clones ($self, $jobs, $comment, $comment_user_id, @clone_args) {
```

Comment:
Shouldn't that be

```suggestion
sub _create_clones ($self, $jobs, $comment = undef, $comment_user_id = undef, @clone_args) {
```

Reply:
No, those arguments will always be passed.

Reply:
Then the first two additional `@job_args` would land in those variables.